### PR TITLE
Submission page

### LINKED ISSUE #69: Submission page
Check firebase if feedback should be viewable
Check python backend if there is an existing submission

image
image

### PR DESCRIPTION
**Pull request by ianmah**
closes #69 

image
image

### CODE REVIEW

**Comment by ianmah** [comment]
Buttons look off but I will wait until #119 is merged to fix

File: src/components/JudgingCard.js
Comment (jackyzha0) [inline src/components/JudgingCard.js:47]: optional params should be last

```suggestion
export default ({ imgUrl, title, teamName, description, buttonLabel, href = '#!' }) => {
```

File: src/components/Submission.js
Comment (jackyzha0) [inline src/components/Submission.js:50]: inline style???

Comment (ianmah) [reply src/components/Submission.js:50]: ehehe i was too lazy to create components for the columns

**Review by jackyzha0 — APPROVED** [review]
thank you king 🧼

**Comment by ianmah** [comment]
no, thank u

**Pull request merged**
